### PR TITLE
rgw: add a few more help options in admin interface

### PR DESCRIPTION
**Pull request by theanalyst**
Added a few missing commandline switches for zone related and orphans related help

### CODE REVIEW

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -197,8 +197,11 @@ void _usage()
   cout << "   --realm-new-name=<realm new name> realm new name\n";
   cout << "   --rgw-zonegroup=<zonegroup>   zonegroup name\n";
   cout << "   --rgw-zone=<zone>         zone in which radosgw is running\n";
+  cout << "   --zone-id=<zone id>       zone id\n";
   cout << "   --zone-new-name=<zone>    zone new name\n";
+  cout << "   --source-zone             specify the source zone (for data sync)";
```
Comment (xiexingguo) [inline src/rgw/rgw_admin.cc]: @theanalyst The false is true, I think you need to append a **newline** here at least.

Comment (theanalyst) [inline src/rgw/rgw_admin.cc]: Ah ok thanks!

**Comment by theanalyst** [comment]
Hm I think whitespacing is an issue in the help file.. causing cli tests to fail.. Let me fix this

**Comment by theanalyst** [comment]
It was basically tabs in the file (that need to be present on a few newlines)

**Comment by xiexingguo** [comment]
:+1:  It's really hard to distinguish these. I think it's better to run UT locally, which shall help to generate the right help.t.

**Comment by theanalyst** [comment]
@xiexingguo yeah actually the problem was my editor stripped lines with empty whitespaces by default, which caused the issue.. now all is well :)

**Pull request merged**